puppetwash.rb

```diff
@@ -152,6 +153,69 @@ def read
   end
 end
 
+# Report relies on end_time and hash. The others are included as useful metadata.
+METADATA_FIELDS = {
+  'end_time': 'string',
+  'environment': 'string',
+  'status': 'string',
+  'noop': 'boolean',
+  'puppet_version': 'string',
+  'producer': 'string',
+  'hash': 'string'
+}
+
+class ReportsDir < Wash::Entry
+  label 'reports_dir'
```

Comment:
@timidri One small nit I had was `facts_dir`; `reports_dir` -- I think the `_dir` is redundant. Are there any issues with changing them to `facts`; `reports` ? I don't think the Wash user will have a hard time figuring out that those are directories.

Reply:
Personally I find the explicit naming convention even easier to follow, but it’s a matter of preference I guess

Reply:
Ok, was just curious.